### ISSUE TITLE
[Feature request] ARM support

### ISSUE THREAD

**Issue by julianfairfax** [39aldo39/Radicale-DecSync#16]
Would be nice if the python module support ARM for the PinePhone and other ARM devices.

**Comment by julianfairfax** [comment]
Sorry for closing and opening this again, would you still be up for adding this functionality? Thanks for your wonderful tweak!

**Comment by julianfairfax** [comment]
@39aldo39 You seem to have been active recently so you might see this. Basically if you install the module for this project, it doesn't actually run on ARM. I don't specifically know which part of your project isn't working, but since it relates directly to my usage of this module, I thought it best I open the issue here. I'm trying to use your project on my Linux phone and I can't do so without ARM support. Thanks for your work on this!

**Comment by 39aldo39** [comment]
Yes, so libdecsync does support ARM now, but the python bindings have a compiled version for x64 included. To get it working I simply have to add an ARM build to it. I will transfer the issue to the right repo.
